static/lang/re-en.json

```
@@ -3281,6 +3281,20 @@
                     "MageArmor": "Scaly Hide w/ Mage Armor"
                 }
             },
+            "DragonForm": {
+                "DragonBreath": {
+                    "Cone":{ 
+                        "Fortitude": "You exhale deadly magical energy in an area, dealing @Damage[(@actor.flags.pf2e.dragonForm.damageDice)d6[(@actor.flags.pf2e.dragonForm.breathDamage)]|options:area-damage] damage to each creature in the area with a @Check[fortitude|against:spell|basic]. The shape is a @Template[type:cone|distance:30]. Once activated, Dragon Breath can't be used again for [[/gmr 1d4 #Recharge Dragon Breath]]{1d4 rounds}.",
+                        "Reflex": "You exhale deadly magical energy in an area, dealing @Damage[(@actor.flags.pf2e.dragonForm.damageDice)d6[(@actor.flags.pf2e.dragonForm.breathDamage)]|options:area-damage] damage to each creature in the area with a @Check[reflex|against:spell|basic]. The shape is a @Template[type:cone|distance:30]. Once activated, Dragon Breath can't be used again for [[/gmr 1d4 #Recharge Dragon Breath]]{1d4 rounds}.",
+                        "Will": "You exhale deadly magical energy in an area, dealing @Damage[(@actor.flags.pf2e.dragonForm.damageDice)d6[(@actor.flags.pf2e.dragonForm.breathDamage)]|options:area-damage] damage to each creature in the area with a @Check[will|against:spell|basic]. The shape is a @Template[type:cone|distance:30]. Once activated, Dragon Breath can't be used again for [[/gmr 1d4 #Recharge Dragon Breath]]{1d4 rounds}."
+                    },
+                    "Line":{ 
+                        "Fortitude": "You exhale deadly magical energy in an area, dealing @Damage[(@actor.flags.pf2e.dragonForm.damageDice)d6[(@actor.flags.pf2e.dragonForm.breathDamage)]|options:area-damage] damage to each creature in the area with a @Check[fortitude|against:spell|basic]. The shape is a @Template[type:line|distance:100]. Once activated, Dragon Breath can't be used again for [[/gmr 1d4 #Recharge Dragon Breath]]{1d4 rounds}.",
+                        "Reflex": "You exhale deadly magical energy in an area, dealing @Damage[(@actor.flags.pf2e.dragonForm.damageDice)d6[(@actor.flags.pf2e.dragonForm.breathDamage)]|options:area-damage] damage to each creature in the area with a @Check[reflex|against:spell|basic]. The shape is a @Template[type:line|distance:100]. Once activated, Dragon Breath can't be used again for [[/gmr 1d4 #Recharge Dragon Breath]]{1d4 rounds}.",
+                        "Will": "You exhale deadly magical energy in an area, dealing @Damage[(@actor.flags.pf2e.dragonForm.damageDice)d6[(@actor.flags.pf2e.dragonForm.breathDamage)]|options:area-damage] damage to each creature in the area with a @Check[will|against:spell|basic]. The shape is a @Template[type:line|distance:100]. Once activated, Dragon Breath can't be used again for [[/gmr 1d4 #Recharge Dragon Breath]]{1d4 rounds}."
+                    }
+                }
+            },
```

```suggestion
            "DragonForm": {
                "DragonBreath": "You exhale deadly magical energy in an area, dealing @Damage[10d6|options:area-damage] damage to each creature in the area with a @Check[{item|flags.pf2e.rulesSelections.dragonbreathSave}|basic|options:area-effect]. The shape is a @Template[type:{item|flags.pf2e.rulesSelections.dragonbreathShape}|distance:{item|flags.pf2e.rulesSelections.dragonbreathDistance}]. Once activated, Dragon Breath can't be used again for [[/r 1d4 #Recharge Dragon Breath]]{1d4 rounds}."
            },
```
We can do away with the actor flags here too by taking a page out of BattleZoo. We use an Item Alteration in the Dragon Form effect to override with this description. You'll need to add another self-resolving Choice Set that defines the distance of the template (`dragonBreathDistance`) but it should work.

This way we don't need to store this many localization strings..